src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -15,25 +15,36 @@
 import frc.robot.hardware.Limelight;
 import frc.robot.utilities.ShooterMath;
 
+/**
+ * A subsystem that's responsible for control of the shooter, either manually or using the
+ * Limelight.
+ */
 public class ShooterSubsystem extends SubsystemBase {
-
-  // Flywheel motors & related devices
+  // Flywheel motors
   private static CANSparkMax m_leftFlywheelMotor =
       new CANSparkMax(kLeftMotorCANId, MotorType.kBrushless);
   private static CANSparkMax m_rightFlywheelMotor =
       new CANSparkMax(kRightMotorCANId, MotorType.kBrushless);
 
+  // The encoder & PID controller only have to be fetched from one motor, since the other follows it
```

Comment:
Up until the point that we get some minute friction differences on one and not the other. Not sure how to deal with that case.

Reply:
Maybe we could characterize the two flywheels separately using SysId in that case? Then we could just control the flywheels independently with their own PID/feedforward constants.

Reply:
Fair. Yeah, we can deal with it when it comes up.